helm/kratos-operator/values.yaml

```diff
@@ -17,6 +17,16 @@ image:
   repository: adobe/kratos
   tag: v0.0.1
 
+prometheus:
+  path: /metrics
```

Comment:
Please check if it /metrics or root endpoint

Reply:
Verified